compiler/modules/front.syntax/parseTypeDesc.bal

```diff
@@ -277,35 +277,29 @@ function parseTypeParam(Tokenizer tok) returns TypeDesc|err:Syntax {
 }
 
 // current token should be "("
-function parseFunctionTypeDesc(Tokenizer tok, FunctionParam[]? fnParams = ()) returns FunctionTypeDesc|err:Syntax {
+function parseFunctionTypeDesc(Tokenizer tok, FunctionTypeParam[] params = []) returns FunctionTypeDesc|err:Syntax {
```

Comment:
Why have you changed the type or the parameter? It was fine. I am saying:

FunctionParams[] type uses a variable `params` ; FunctionTypeParams[] type uses a variable `typeParams`.

Reply:
fixed